src/index.ts

```diff
@@ -37,19 +45,35 @@ export default class BrowserTransport implements LinkTransport {
         this.classPrefix = options.classPrefix || 'anchor-link'
         this.injectStyles = !(options.injectStyles === false)
         this.requestStatus = !(options.requestStatus === false)
+        this.fuelEnabled = options.enableGreymassFuel === true
         this.storage = new Storage(options.storagePrefix || 'anchor-link')
+        this.prepareStatus = 'Preparing request...'
     }
 
     private classPrefix: string
     private injectStyles: boolean
     private requestStatus: boolean
+    private fuelEnabled: boolean
     private activeRequest?: SigningRequest
     private activeCancel?: (reason: string | Error) => void
     private containerEl!: HTMLElement
     private requestEl!: HTMLElement
     private styleEl?: HTMLStyleElement
     private countdownTimer?: NodeJS.Timeout
     private closeTimer?: NodeJS.Timeout
+    private prepareMessageTimer?: NodeJS.Timeout
```

Comment:
Why use a timer for this?

Reply:
Hrm, I was having a difficult time getting the UI to update dynamically so I mirrored how the other dynamic components were doing it. I'll try again to see if I can remove the timer.

Reply:
Swapped this over to using a HTMLElement on this as opposed to a timer.